### PR TITLE
graC4C-130: Added a Login and Register page to the views of the web app.…

### PR DESCRIPTION
**Pull request by kubowitsche**

### CODE REVIEW

File: C4CCoreApp/C4CAndroid/AddonsActivity.cs [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace C4CAndroid
+{
+    [Activity(Label = "AddonsActivity")]
+    public class AddonsActivity : Activity
+    {
+
+        //TODO add price to this screen
```
Comment (hackeret) [inline C4CCoreApp/C4CAndroid/AddonsActivity.cs]: It appears price was added to the screen is this todo still valid?

File: C4CCoreApp/C4CAndroid/Resources/layout/Checkout.axml [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/textView1"
+                android:textColor="@color/dark_lavender" />
+            <TextView
+                android:text="Who Is This Box Being Sent To?"
+                style="@style/text_medium"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/textView2"
```
Comment (hackeret) [inline C4CCoreApp/C4CAndroid/Resources/layout/Checkout.axml]: This occurs throughout our axmls but we should pay attention to making these id's more explicit

File: C4CCoreApp/C4CAndroid/AddonsActivity.cs (hunk 1 above)
Comment (kubowitsche) [reply C4CCoreApp/C4CAndroid/AddonsActivity.cs]: I did that as part of the migration. I've removed this comment

File: C4CCoreApp/C4CAndroid/Resources/layout/Checkout.axml (hunk 2 above)
Comment (kubowitsche) [reply C4CCoreApp/C4CAndroid/Resources/layout/Checkout.axml]: Probably a good idea but I don't think it has to be high priority right now

**Pull request merged**